MAINTAINERS.md

```diff
@@ -0,0 +1,21 @@
+# Maintainers
+
+Maintainers are responsible for the overall health of the project. They are responsible for reviewing issues, pull requests, and ensuring that the project is moving in the right direction.
+
+## Pull Requests - Merging Requirements
+
+To ensure everyone in the community has the opportunity to share their feedback, ideas, and suggestions, a Pull Request must go through a review process, and must meet the following requirements to be merged:
+
+- **Approval:** At least **2 core team members** must approve the Pull Request.
```

Comment:
Honestly, rules are made to be written and read.

I don't expect to see them applied without  common sense.

So I would say this value is OK for someone reading it, so they understand the workflow and expectations.

Then PR will be merged when needed.

But maybe my visiot is this one because I'm French and you are German 😂😅

Reply:
The strict German in me says "two are set in stone now" ;) But the other half says: the total number of team members is neither specified nor relevant - so maybe this is also a partial encouragement to create a "reviewers" role with people like @ccoVeille and me, who eagerly review code and have different views and experiences.

In the end I really would like to encourage a four-eyes principle - simple because I would not be sure that I always grasp 100% of it, no matter how much I try.

Reply:
> as the team is still pretty small

Not that small. :smile: We're 9 members in the organization, and if we add @klaernie as a maintainer, we're 10 (I definitely agree to add klaernie to the organization/maintainer team).
I think, we'll need to try, and if we see it's "too much", we will be able to reduce this number, to only 1 approval.

On this PR, there are already 5 reviewers, so I feel like, it's working pretty well, and we'll be able at least to get 2 reviews per PR, and if not, because people don't have time to review "right away", it can wait a few days/weeks. I don't think we really need to urge for changes, we can wait, as editorconfig-checker is a pretty mature software already.

Reply:
Ah, okay, that number of maintainers is surprising, given that the org only lists four (as they made their role known to the world). With that number 2 is a good number of reviews to require.